### ISSUE TITLE
Integrate a Crawler to crawl data from the links sent as a message

### ISSUE THREAD

**Issue by ankitjain28may** [ankitjain28may/openchat#32]
Crawler crawls some basic data show the user a basic outline of the link sent in the message.

**Comment by kniziol** [comment]
Where/How to start? Where to integrate the Crawler? Would you like to implement [symfony/dom-crawler](https://packagist.org/packages/symfony/dom-crawler)? :)

**Comment by peter279k** [comment]
I think using `dom-crawler` is fine.

What do you think about that, @ankitjain28may :) ?

**Comment by ankitjain28may** [comment]
@kniziol @peter279k Yeah we can use `dom-crawler`, just need the title and description to the page similar to telegram and other messaging platforms.

**Comment by kniziol** [comment]
Where/How to start? Where to integrate the Crawler?
